ft_memmove.c

```diff
@@ -0,0 +1,23 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_memmove.c                                       :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ekorley <marvin@42.fr>                     +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2016/09/22 16:32:37 by ekorley           #+#    #+#             */
+/*   Updated: 2016/09/22 17:15:50 by ekorley          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+
+void	*ft_memmove(void *dest, const void *src, size_t n)
+{
+	unsigned char	*temp;
+
+	temp = (unsigned char *)malloc(n);
+	ft_memcpy(temp, src, n);
+	ft_memcpy(dest, temp, n);
+	return (dest);
+}
```

Comment:
I'm not familiar with this function, but if you can wait on the review I'll get back to you.